src/main/java/jenkins/plugins/slack/pipeline/SlackUserIdsFromCommittersStep.java

```diff
@@ -0,0 +1,175 @@
+package jenkins.plugins.slack.pipeline;
+
+import com.google.common.collect.ImmutableSet;
+import hudson.Extension;
+import hudson.ProxyConfiguration;
+import hudson.Util;
+import hudson.model.Item;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.util.ListBoxModel;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import jenkins.model.Jenkins;
+import jenkins.plugins.slack.CredentialsObtainer;
+import jenkins.plugins.slack.HttpClient;
+import jenkins.plugins.slack.Messages;
+import jenkins.plugins.slack.SlackNotifier;
+import jenkins.plugins.slack.user.SlackUserIdResolver;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution;
+import org.kohsuke.stapler.AncestorInPath;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import static jenkins.plugins.slack.CredentialsObtainer.getItemForCredentials;
+import static jenkins.plugins.slack.SlackNotifier.DescriptorImpl.findTokenCredentialIdItems;
+
+/**
+ * Workflow step to send a Slack channel notification.
+ */
+public class SlackUserIdsFromCommittersStep extends Step {
+
+    private static final Logger logger = Logger.getLogger(SlackUserIdsFromCommittersStep.class.getName());
+
+    private String token;
```

Comment:
Can you remove token from both steps please and just use the credential id, It’s more secure, stops the token being logged in the build by accident

Reply:
Oh I totally missed the token 😰

Reply:
> Can you remove token from both steps please and just use the credential id, It’s more secure, stops the token being logged in the build by accident
@timja Fixed in [52bd358](https://github.com/jenkinsci/slack-plugin/pull/669/commits/52bd358223aaeaa83879c69b7cd610f85eb5b0c6)